.github/workflows/upload.yml

```diff
@@ -19,7 +19,6 @@ jobs:
         npm run build --if-present
         npm run test --if-present
     - uses: Azure/functions-action@v1
-      id: fa
```

Comment:
This isn't needed, because no other steps use the id

Reply:
Yeah, I was wondering if it can be dropped too...